mne/epochs.py

```diff
@@ -2990,7 +2990,7 @@ def _ensure_list(x):
 
     # keep_first and keep_last names
     start_idx = stop_idx
-    metadata.iloc[:, start_idx:] = None
+    metadata[columns[start_idx:]] = ""
```

Comment:
I'm not sure if this is guaranteed to work in-place (now and in the future). The cleaner approach for this specific case here is definitely through using `.loc` or `iloc`

Reply:
I don't think that this is a problem here; usually, the infamous `SettingWithCopyWarning` appears only after chaining two or more indexing operations (https://pandas.pydata.org/docs/user_guide/indexing.html#indexing-view-versus-copy).

Also, the previous line triggered the `FutureWarning` – do you have an alternative suggestion how to typecast the column (to `object`) and then set its entries to `None`?

Reply:
In case I'm wrong, do you have a link for a list of channel labels not being safe?

Reply:
Ok let's keep it, then :)